### PR TITLE
Updated emission factors CL-SEN (2022)

### PR DESCRIPTION
**Pull request by florianscheidl**
## Issue

ref [ELE-2974](https://linear.app/electricitymaps/issue/ELE-2974/update-emission-factors-in-chile-cl-sen)

1. The unknown category of the Chile SEN emission factors accounted for all production modes, while it seems they should only include thermal production models, see [this](https://github.com/electricitymaps/electricitymaps-contrib/blob/40b441ec727e44958b772bbb103873c37f1c2a6e/config/zones/CL-SEN.yaml#L23) disclaimer. 
2. Most recent emission factors were from 2021.

## Description

1. Added emission factors for 2022,
2. added a [sheet](https://docs.google.com/spreadsheets/d/1-iXUGhAM_9Ft5ejlNVxaqaAwG8Octh3gSjU-dm4e96o/edit#gid=163148677) in the source document.
3. In this source document, pulled and pasted the most recent data from the source, recomputed thermal emission factors (direct and lifecycle) in the sheet
4. copied updated factors to the config file, updated the source reference.

### CODE REVIEW

File: config/zones/CL-SEN.yaml [hunk 1]
```diff
@@ -71,19 +71,22 @@ emissionFactors:
...
         source: Electricity Maps, CEN
-        value: 349.6068089
+        value: 602.5343343
       - datetime: '2021-01-01'
         source: Electricity Maps, CEN
-        value: 351.9679562
+        value: 597.8259335
+      - datetime: '2022-01-01'
+        source: Electricity Maps, CEN
+        value: 556.6130553
```
[the remark this answers is not in the document]
Comment (VIKTORVAV99) [reply config/zones/CL-SEN.yaml:89]: I would advise against this, not all platforms and YAML parsers support this.

File: config/zones/CL-SEN.yaml [hunk 2]
```diff
@@ -257,5 +263,5 @@ parsers:
   productionCapacity: CL_SEN.fetch_production_capacity
 sources:
   Electricity Maps, CEN:
-    link: https://docs.google.com/spreadsheets/d/1-iXUGhAM_9Ft5ejlNVxaqaAwG8Octh3gSjU-dm4e96o/edit#gid=0
+    link: https://docs.google.com/spreadsheets/d/1-iXUGhAM_9Ft5ejlNVxaqaAwG8Octh3gSjU-dm4e96o/edit#gid=163148677 (average thermoelectric emission factors)
```
Comment (q--) [inline config/zones/CL-SEN.yaml:266]: The spreadsheet is not publicly accessible, is this intentional?

Comment (VIKTORVAV99) [reply config/zones/CL-SEN.yaml:266]: Try now, I updated the permissions. I believe all our sources are intended to be public.

Comment (q--) [reply config/zones/CL-SEN.yaml:266]: Fixed, thank you!

**Review by mathilde-daugy — APPROVED** [review]

**Comment by mathilde-daugy** [comment]
Thanks for taking care of this @florianscheidl ! :)

**Pull request merged**